next.config.js

```diff
@@ -1,16 +1,40 @@
 /** @type {import('next').NextConfig} */
+require('webpack')
+require('path')
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { withPlausibleProxy } = require('next-plausible')
 
 module.exports = withPlausibleProxy()({
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
Fine to change to `false` temporarily, but you will need to reenable this again when you finish up this PR.